File: DesktopSample/DataBinding/Section/IListBinding/VB.NET/IListBinding/ViewerForm.vb
Imports GrapeCity.ActiveReports

Public Class ViewerForm
	Inherits System.Windows.Forms.Form

	Public Sub New()
		MyBase.New()

		'Windows フォーム デザイナ サポートに必要です。
		InitializeComponent()
	End Sub

	'使用されているリソースをクリーンアップします。
	Protected Overloads Overrides Sub Dispose(ByVal disposing As Boolean)
		If disposing Then
			If Not (components Is Nothing) Then
				components.Dispose()
			End If
		End If
		MyBase.Dispose(disposing)
	End Sub

	Public Sub LoadReport(ByVal report As SectionReport)
		arvMain.LoadDocument(report)
	End Sub
End Class

File: DesktopSample/DataBinding/Section/IListBinding/C#/IListBinding/ViewerForm.cs

using GrapeCity.ActiveReports;

namespace ActiveReports.Samples.IListBinding
{
	/// <summary>
	/// ViewerForm の概要の説明です。
	/// </summary>
	public partial class ViewerForm : System.Windows.Forms.Form
	{
		public ViewerForm()
		{
			//
			// Windows フォーム デザイナ サポートに必要です。
			//
			InitializeComponent();
		}

		public void LoadDocument(SectionReport rpt)
		{
			arvMain.LoadDocument(rpt);
		}
	}
}
